fendermint/vm/genesis/src/lib.rs

```diff
@@ -92,6 +94,17 @@ pub struct Collateral(#[serde_as(as = "IsHumanReadable")] pub TokenAmount);
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Copy)]
 pub struct Power(pub u64);
 
+impl Power {
+    /// Convert from [Power] to [Collateral] by specifying the number of significant
+    /// decimal places per FIL that grant 1 power.
+    pub fn into_collateral(self: Power, scale: PowerScale) -> Collateral {
```

Comment:
I don't think we should have a function like this. The power scale works one way, but it's a lossy transformation, we can't expect it go back and be the same as the collateral. It should work without this.